dataservice/api/family/schemas.py

```diff
@@ -1,9 +1,16 @@
+from marshmallow_sqlalchemy import field_for
 from dataservice.api.family.models import Family
 from dataservice.api.common.schemas import BaseSchema
 from dataservice.extensions import ma
+from dataservice.api.common.validation import (enum_validation_generator)
+FAMILY_TYPE_ENUM = {"Proband Only", "Duo", "Duo+",
+                    "Trio", "Trio +", "Other", }
```

Comment:
Should probably be `Trio+` right? Looks like theres a space right now between Trio and +